### PR TITLE
Long text goes outside cards

### PR DESCRIPTION
**Pull request by Link2Twenty**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Add word-break to crayons-card this will allow words to be broken if the word is too long but will wrap normally otherwise.

## Related Tickets & Documents
closes: #10629

## QA Instructions, Screenshots, Recordings
forem__resize-bug

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

fix it with love

### CODE REVIEW

File: app/assets/stylesheets/components/cards.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
   background: var(--card-bg);
   color: var(--card-color);
   box-shadow: 0 0 0 1px var(--card-border);
+  word-break: break-word;
```
Comment (reobin) [inline app/assets/stylesheets/components/cards.scss:8]: Hey @Link2Twenty !

Just dropping [this (mdn word-break docs)](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) here.

Quotes about `word-break: break-word`:
* "This deprecated API should no longer be used, but will probably still work"
* Solution: "Has the same effect as `word-break: normal` and `overflow-wrap: anywhere`, regardless of the actual value of the `overflow-wrap` property."

Comment (Link2Twenty) [reply app/assets/stylesheets/components/cards.scss]: Good catch 😅

**Comment by ludwiczakpawel** [comment]
I'll play with it locally since it touches `.crayons-card` and that being used like everywhere  :D. 

Also out of curiosity, why not `overflow-wrap: break-word;`?

**Comment by Link2Twenty** [comment]
No real reason, all you gain with everywhere is the ability to use `min-content` and `max-content` as widths.

[example](https://jsfiddle.net/link2twenty/Lyeaxmk3/)

**Review by ludwiczakpawel — APPROVED** [review]
Cheers!

**Review by benhalpern — APPROVED** [review]

**Pull request merged**